pom.xml

```diff
@@ -43,8 +43,8 @@
     <spring.boot.version>2.3.3.RELEASE</spring.boot.version>
     <jackson.version>2.11.2</jackson.version>
     <!-- metadb messaging and shared entities dependency versions -->
-    <cmo_metadb_messaging_java.group>com.github.mskcc</cmo_metadb_messaging_java.group>
-    <cmo_metadb_messaging_java.version>81a77db772315f2f0e47f3eeff8fabd0d06848e4</cmo_metadb_messaging_java.version>
+    <cmo_metadb_messaging_java.group>com.github.ao508</cmo_metadb_messaging_java.group>
```

Comment:
will need to change the commit and group id to the upstream repo once that pr is merged